File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/EnrollmentRequest.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class EnrollmentRequest : DomainResource, IResource
{
    public string? Status { get; set; }
    public string? Created { get; set; }
    public Base.ResourceReference? Insurer { get; set; }
    public Base.ResourceReference? Coverage { get; set; }
    public Base.ResourceReference? Provider { get; set; }
    public Base.ResourceReference? Candidate { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/EnrollmentRequest.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class EnrollmentRequest(DomainResource):
    status: Optional[str] = None
    created: Optional[str] = None
    insurer: Optional[Reference] = None
    coverage: Optional[Reference] = None
    provider: Optional[Reference] = None
    candidate: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)